plotnine/tests/test_geom_text_label.py

```diff
@@ -49,3 +49,37 @@ def test_label_aesthetics():
          scale_y_continuous(limits=(-0.5, n-0.5)))
 
     assert p == 'label_aesthetics'
+
+
+def test_adjust_text():
+    from plotnine.data import mtcars
+    p = (
+        ggplot(
+            mtcars.tail(2),  # two points -> fast test.
+            aes('mpg', 'disp', label='name')) + geom_point(
+                size=5, fill='black') + geom_text(
+                        adjust_text={
+                            'expand_points': (2, 2),
+                            'arrowprops': {
+                                'arrowstyle': '->',
+                                'color': 'red'
+                            }
+                        }))
```

Comment:
Can you change the styling for the test to something like

```python
    adjust_text={
        'expand_points': (2, 2),
        'arrowprops': {
            'arrowstyle': '->',
            'color': 'red'
        }
    }

    p = (ggplot(mtcars.tail(2), aes('mpg', 'disp', label='name'))
         + geom_point(size=5, fill='black')
         + geom_text(adjust_text=adjust_text)
         )
```

That way, it is easy to tell which layers are in the plot and also to comment out any layer. In some of the earlier tests the `+` signs are on the right (not to my exact preference) since the pep8 standard (and the linting tools) was broken, but it has since been [fixed](https://github.com/python/peps/commit/7690a00d478866eb3cae3d82bbe1e80199a7392f#diff-64ec08cc46db7540f18f2af46037f599).